src/Network/Ethereum/Web3/Solidity/Generic.purs

```diff
@@ -201,39 +203,51 @@ instance argsToRowListProxyBase :: ArgsToRowListProxy (Argument (Tagged (SProxy
 instance argsToRowListProxyInductive :: ArgsToRowListProxy as l => ArgsToRowListProxy (Product (Argument (Tagged (SProxy s) a)) as) (Cons s a l) where
   argsToRowListProxy _ = RLProxy
 
-class ToRecordFields args fields (rowList :: RowList) | args -> rowList, rowList -> args fields where
+class ListToRow rowList fields <= RecordFieldsIso args fields (rowList :: RowList) | args -> rowList, rowList -> args fields where
```

Comment:
I think the kind annotation (` :: RowList`) it's not needed, as it should be Inferred from `ListToRow`